gulp-tasks/jekyll.js

```diff
@@ -66,14 +52,22 @@ const runJekyllCommand = (command, additionalParams) => {
   });
 };
 
-gulp.task('jekyll:build', () => {
-  return runJekyllCommand('build');
+gulp.task('npm-dependencies', () => {
+  return gulp.src([
+    path.join(NODE_MODULES, 'anchor-js', 'anchor.min.js'),
```

Comment:
I hate to do this to you, but now that these are passed to `gulp.src()`, you actually **need** to use `/`, even on Windows. They're interpreted as glob patterns, and they won't work with `\`.

https://github.com/isaacs/node-glob#windows